Tests/Test_BSP_Lights.c

```diff
@@ -0,0 +1,32 @@
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+int main() {
+    BSP_Lights_Init();
+    printf("M_CNCTR\tRSVD_LED\tCTRL_FAULT\tRIGHT_BLINK\tHeadlightPWR\tA_CNCTR\tBPS_FAULT\tLEFT_BLINK\tBPS_PWR\tBrakeLight\n");
+    while(1) {
+        printf("%d\t%d\t%d\t%d\t%d\t%d\t%d\t%d\t%d\t%d\r", 
```

Comment:
The values aren't lined up under their respective labels, they should probably be more spaced out.